### PR TITLE
You can get more money from req

### PR DESCRIPTION
**Pull request by novaepee**
## About The Pull Request

Title

## Why It's Good For The Game

image

## Changelog

:cl:
add: You can get more money from req
/:cl:

### CODE REVIEW

**Review by ivanmixo — CHANGES_REQUESTED** [review]
Move it under supplies, it's the de-facto fluff tab.

File: code/modules/reqs/supplypacks.dm [hunk 1]
```diff
@@ -148,6 +143,36 @@ OPERATIONS
 	containertype = null
 	cost = 100
 
+/datum/supply_packs/operations/c10
```
Comment (TiviPlus) [inline code/modules/reqs/supplypacks.dm:146]: No numbers in typepaths pls

**Review by TiviPlus — CHANGES_REQUESTED** [review]

**Comment by Pariah919** [comment]
inflation has truly defined the terragov economy

File: code/modules/reqs/supplypacks.dm [hunk 2]
```diff
@@ -1464,6 +1434,37 @@ SUPPLIES
...
+	name = "100 dollars"
+	contains = list(/obj/item/spacecash/c100)
+	cost = 10
+
+/datum/supply_packs/operations/dollartwohundred
+	name = "200 dollars"
+	contains = list(/obj/item/spacecash/c200)
+	cost = 20
+
+/datum/supply_packs/operations/dullarfivehundred
```
Comment (ivanmixo) [inline code/modules/reqs/supplypacks.dm:1463]: `dullar`. Also move it to supplies, you didn't change the path

**Review by ivanmixo — CHANGES_REQUESTED** [review]

File: code/modules/reqs/supplypacks.dm (hunk 2 above)
Comment (novaepee) [reply code/modules/reqs/supplypacks.dm:1463]: truly the strongest currency in the universe

**Comment by chizzy376** [comment]
fluff PR? for what reason?

**Review by ivanmixo — APPROVED** [review]

**Pull request merged**